### DISCUSSION TITLE
Parts and their replacements

### DISCUSSION THREAD

**Discussion by superryan** [virtualagc/virtualagc#1216] [Q&A]
Good day,

I have only basic understanding of electronics, so some obvious things might got past me.
In time I'm aiming to build an AGC myself. However, to understand the basics of it's behavior I want to make one on breadboards first.
Looking at some parts however I can't seem to find a comparable replacement on current market.
I have seen that there are others who have the same idea of making one, and was wondering if I could tab into their knowledge when building one myself.

When I was discussing this idea with some co workers who do electronics they told me I need to the specific part number of the transistor I want to use, and that just 3 characteristics would not be enough like with the 2004004 which I found in the power supply (2005916) of the computer. They told me that I not only would need to know the Vceo, Vcbo and Vebo, but also the bandwidth. However, I do not recognize it in the drawing.

What would in this case be the specifics I would need to look for?

P.S. I'm going to keep asking a lot of these questions likely until I understand enough electronics that I can do more than just follow a drawing.

Sincerely, Ryan

**Comment by thewonderidiot** [comment]
I don't have a good answer for your bandwidth question, but here's a few tidbits that may or may not be helpful:

- In case you didn't pick it intentionally, 2005916 is the schematic for an earlier revision of the power supply module. 2005971 is for the version that ended up flying.
- You need to be specific with dash numbers for parts. 2004004 can either be an NPN (mapping to 1006323) or a PNP (mapping to 1006310), depending on its dash number.
- I've never found anything that states it directly, but I've long suspected that 1006323 is either what we now know as 2N2222, or something extremely closely related to it. [This report from MIT](https://www.ibiblio.org/apollo/Documents/Apollo%20Report%20E-2218.pdf) and [this report from Motorola](https://ntrs.nasa.gov/api/citations/19730009742/downloads/19730009742.pdf) describe essentially exactly the same problem, down to all of the materials and the physical construction. The former only mentions the 1006323 transistor, while the latter only mentions 2N2222. But it looks like the same thing to me. If that is the case, then presumably 1006310 would be 2N2907 or something along those lines.

**Comment by superryan** [comment]
I did not pick the older version intentionally. I aim to build an AGC as flown in Apollo 17(or 18 if they ever got there). Thank you for mentioning this.

Looking at the 2005971 I see they used different transistors. The 2005916 used 2004004-001 and the 2005971 used 2004184-001.
Were those similar in behavior or were there differences needed after the removal  of the +28COM and WD243/WD355 pins?

**Comment by thewonderidiot** [comment]
2004004-001 and 2004184-001 are actually the same transistor -- both of them are encapsulated flight-qualified 1006323 transistors.

From [drawing 2004004](https://archive.org/details/apertureCardBox459NARASW_images/apertureCardBox459NARASW/page/n1007/mode/1up?view=theater):
![image](https://github.com/virtualagc/virtualagc/assets/94056/017de763-5476-4e97-8b3c-09f1ac72a11c)

From [drawing 2004184](https://archive.org/details/apertureCardBox459NARASW_images/apertureCardBox459NARASW/page/n1386/mode/1up?view=theater):
![image](https://github.com/virtualagc/virtualagc/assets/94056/0ab63eab-59ab-4eb6-93c9-9b744a0fad56)

**Comment by superryan** [comment]
Thank you for answering.
Also thank you for all the research you have done and willing to share all the info^^

**Comment by dak4482** [comment]
Here's the link to the actual transistor data
https://archive.org/details/apertureCardBox439Part2NARASW_images/page/n299/mode/1up?view=theater
scroll down to page 302
